front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -42,7 +48,45 @@ const ManchetteWithSpaceTimeChartWrapper = ({
   );
 
   const [showSettingsPanel, setShowSettingsPanel] = useState(false);
-  const [settings, setSettings] = useState({ showConflicts: false });
+  const [settings, setSettings] = useState({
+    showConflicts: false,
+    showBAL: false,
+    showBAPR: false,
+    showTVM: false,
```

Comment:
As @flomonster noted, could we avoid hardcoding particular signalling systems?

Reply:
We can use `/sprites/signaling_systems`

Reply:
I got rid of it, fixed

Reply:
There's a single checkbox now so we don't need this anymore.